package.json

```
@@ -13,36 +13,170 @@
     "node": ">=10.0.0"
   },
   "scripts": {
+    "prepare": "husky install",
     "build": "rimraf dist && rollup -c rollup.config.js",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
     "lint:fix": "yarn lint --fix",
     "test": "jest",
     "generateTypes": "yarn openapi-typescript https://fingerprintjs.github.io/fingerprint-pro-server-api-openapi/schemes/fingerprint-server-api.yaml --output ./src/generatedApiTypes.ts -c .prettierrc.json",
-    "release:major": "changelog -M && git add CHANGELOG.md && yarn version --major --no-git-tag-version",
-    "release:minor": "changelog -m && git add CHANGELOG.md && yarn version --minor --no-git-tag-version",
-    "release:patch": "changelog -p && git add CHANGELOG.md && yarn version --patch --no-git-tag-version"
+    "release": "semantic-release"
   },
   "keywords": [],
-  "author": "FingerprintJS, Inc (https://fingerprintjs.com)",
+  "author": "Fingerprint, Inc (https://fingerprint.com)",
```

```suggestion
  "author": "FingerprintJS, Inc (https://fingerprint.com)",
```

legal name didn't change